Update node table upload info + typo fixes (#13)
multinet-app/multinet-docs: multinet.rst

as a social network connecting the various Revolutionary War figures to the
social clubs they belonged to.

The first step for any session where you're modifying data, is to log in using
Google OAuth. Click the person icon next to the multinet logo and then click 
'Sign in with Google'. Follow the prompts until you're back on the multinet site.

> Authenticating allows you to upload data into certain workspaces (this is based 
> on your permissions level).

Reviewer: Better to keep this as simple as possible:

```suggestion
Authenticating allows you to create workspaces and upload data to them.
```
